src/app/shared/components/navbar/navbar.component.html

```diff
@@ -27,9 +27,13 @@
           }}</button>
       </li>
       <li class="nav-item">
-        <button onclick="window.location.href = '/team'" class="buttonNavbar" type="button">{{ 'NAVBAR.TEAM' | translate
-          }}</button>
+        <button class="buttonNavbar" type="button" [matMenuTriggerFor]="institutionalMenu">{{ 'NAVBAR.INSTITUTIONAL' | translate }}</button>
       </li>
+      <mat-menu #institutionalMenu="matMenu" xPosition="before" yPosition="below">
+        <button mat-menu-item onclick="window.location.href = '/team'">{{ 'NAVBAR.TEAM' | translate }}</button>
+        <button mat-menu-item onclick="window.location.href = 'https://csitba.web.app/'">CS</button>
+        <button mat-menu-item onclick="window.location.href = 'https://linktr.ee/wie_itba'">WIE</button>
```

Comment:
Acá no usaría onclick, los botones traen properties específicas que sirven para eso sin sobrescribir la lógica de su click. Por si las dudas chequeá que te funcione
Para las rutas dentro de nuestra app usaría: routerLink="/team" y fletaría la otra property
Para las rutas fuera de nuestra app usaría href="URL_GOES_HERE"
```suggestion
        <button mat-menu-item routerLink= "/team">{{ 'NAVBAR.TEAM' | translate }}</button>
        <button mat-menu-item href="https://csitba.web.app/">CS</button>
        <button mat-menu-item href="https://linktr.ee/wie_itba">WIE</button>
```